### ISSUE TITLE
PLEASE add the option to migrate layers styles to color variables

### ISSUE THREAD

**Issue by downtowndesign** [sketch-hq/color-variables-migrator#2]
The much more often case is to migrate layer styles to color variables!

**Comment by jujemido** [comment]
Yes please.... we are having a lot of trouble and forced to roll back to v68 due to this and the shared libraries problem with abstract
